crates/ika-move-packages/packages/ika_system/sources/dwallet/dwallet_2pc_mpc_coordinator_inner.move

```diff
@@ -631,6 +675,12 @@ public struct ECDSASignRequestEvent has copy, drop, store {
     /// The output from the dWallet DKG process used in this session.
     dwallet_public_output: vector<u8>,
 
+    /// The elliptic curve used for the dWallet.
+    curve: u8,
```

Comment:
Just saying it: using a `u8` limits us to `256` supported options, forever. This should be fine, but I'm wondering why not use a bigger number like `u32` which we know never, ever to be too small for our future needs. I guess maybe storage is an reason, but its probably negligible relative to other fields.